docs/dashboards/style-by-css.md

```diff
@@ -0,0 +1,251 @@
+files that you need.
+
+## General classes
+Each of the class name contains a prefix `highcharts-dashboards` and a suffix that
+describes the element.   
+For example, the class name for the dashboard's row is `highcharts-dashboards-row`.
+
+We can distinguish a few main elements and their classes:
+- `highcharts-dashboards` - the main class for the dashboard
+- `highcharts-dashboards-row` - the class for the dashboard's row
+- `highcharts-dashboards-cell` - the class for the dashboard's cell
+- `highcharts-dashboards-component` - the class for the dashboard's component
+
+The rest of the classes are specific for each component or element.
+
+To access and style the whole dashboard, its background and component you can use:
+```css
+.highcharts-dashboards,
+.highcharts-dashboards-wrapper {
+    background-color: #f2f9fd;
+}
+```
+
+To style the dashboard's row:
+```css
+.highcharts-dashboards-row {
+    padding: 10px;
+}
+```
+
+To style the dashboard's cell:
+```css
+.highcharts-dashboards-cell {
+    text-align: left;
+}
+```
+
+To style the dashboard's component:
+```css
+.highcharts-dashboards-component {
+    border-radius: 10px;
+}
+```
+
+## Component classes
+Each component has the generic component CSS class `highcharts-dashboards-component`
+and its own CSS class specific for the component. For example, the KPI component
+has the class `highcharts-dashboards-component-kpi`.
+
+To style the dashboard's component title:
+```css
+.highcharts-dashboards-component-title {
+    font-size: 12px;
+}
+```
+
+To style the dashboard's component subtitle:
+```css
+.highcharts-dashboards-component-subtitle {
+    font-size: 10px;
+}
+```
+
+### KPI Component
+To style the dashboard's KPI component:
+```css
+.highcharts-dashboards-component-kpi {
+    border-radius: 10px;
+}
+```
+
+To style the dashboard's KPI value:
+```css
+.highcharts-dashboards-component-kpi-value {
+    color: red;
+}
+```
+
+### Highcharts Component
+To style the dashboard's Highcharts component:
+```css
+.highcharts-dashboards-component-highcharts {
+    background-color: gray;
+}
+```
+
+To individually style the dashboard's Highcharts chart please find more information on how to [style Highcharts Chart.](https://www.highcharts.com/docs/chart-design-and-style/style-by-css)
+
+### DataGrid Component
+To style the dashboard's DataGrid component:
+```css
+.highcharts-datagrid-container {
+    border-radius: 10px;
+}
+```
+
+To style the dashboard's DataGrid header:
+```css
+.highcharts-datagrid-column-header {
+    background-color: gray;
+}
+```
+
+To style the dashboard's DataGrid row:
+```css
+.highcharts-datagrid-row {
+    background-color: gray;
+}
+```
+
+Note that you can use the child selector to style even and odd rows:
+```css
+.highcharts-datagrid-row:nth-child(even) {
+    background-color: white;
+}
+```
+
+### HTML Component
+Since this component is based on the HTML element and can be very different
```

Comment:
All of the components are based on the HTML elements, so saying this sentence here does not make much sense. I'd go for something like `Since whole structure of the HTML element is defined by the user ...`